lnrpc/routerrpc/router.proto

```diff
@@ -213,6 +213,43 @@ message ResetMissionControlRequest{}
 
 message ResetMissionControlResponse{}
 
+message QueryMissionControlRequest {}
+
+/// QueryMissionControlResponse contains mission control state per node.
+message QueryMissionControlResponse {
+    repeated NodeHistory nodes = 1;
+}
+
+/// NodeHistory contains the mission control state for a particular node.
+message NodeHistory {
+    /// Node pubkey
+    bytes pubkey = 1;
+
+    /// Time stamp of last failure. Set to zero if no failure happened yet.
+    int64 last_fail_time = 2;
```

Comment:
`_unix` suffix perhaps?

Reply:
I don't know. I think we may first want to discuss what our standards should be regarding suffixes. It isn't consistent at the moment.

Reply:
These fields can always be renamed without breaking backwards compatibility

Reply:
idt we have a standard for timestamps. note that renaming the field does break API compatibility, but not serialization

Reply:
Oh, that is true indeed. If people rebuild their apps with the new proto, it will break.